server/views/taskList/taskListTemplate.pug

```diff
@@ -33,6 +33,15 @@ block content
   if !user.role
     p Error
   else
+    if errors
+    div.largeMarginBottom
+    if errors && errors.length !== 0
```

Comment:
once not relying on null to represent "no errors", we can remove this check

Reply:
and by passing through empty array - we don't need to handle the null check here either.

Reply:
done